packages/mds-providers/index.ts

```diff
@@ -269,15 +273,24 @@ export const providers: Readonly<{ [P in PROVIDER_ID]: Readonly<Provider> }> = O
   })
 })
 
+export const getProviders = async (): Promise<Readonly<{ [P in PROVIDER_ID]: Readonly<Provider> }>> => {
+  return providers
+}
 const isProviderId = (provider_id: unknown): provider_id is PROVIDER_ID => {
   return typeof provider_id === 'string' && providers[provider_id as PROVIDER_ID] !== undefined
 }
 
 /**
+ * TODO replace with real mds-providers service
  * convert provider_id to provider name (if available), or a subset of the UUID for humans
  * @param  provider_id
  * @return name or provider_id substring
  */
-export const providerName = (provider_id: string): string => {
-  return isProviderId(provider_id) ? providers[provider_id].provider_name : provider_id
+export const providerName = async (provider_id: string): Promise<string> => {
+  const name = isProviderId(provider_id) ? providers[provider_id].provider_name : provider_id
+  return new Promise((resolve, reject) => {
+    setTimeout(() => {
+      resolve(name)
```

Comment:
Could you just return Promise.resolve(name)? Why the setTimeout?

Reply:
May not even need to do that, can't you mark something as returning a promise even if it just returns a value directly?

Like:
```
const x = async (): Promise<number> =>  5
```

Reply:
I thought it'd be a better validation of the earlier `Promise.all` calls being correct if they were truly asynchronous.